lib/morse_translator.rb

```diff
@@ -1,6 +1,8 @@
-class MorseTranslator
 
-  def initialize
+
+class MorseTranslator
+  attr_reader :dictionary
+  def initialize(dictionary)
```

Comment:
You do not need to pass the dictionary in to the initialize method - you can just initialize with the dictionary as the default value that was given.